frog/imports/ui/GraphEditor/utils/export.js

```diff
@@ -34,10 +35,11 @@ const doImportGraph = graphStr => {
   const graphObj = JSON.parse(graphStr.target.result);
   const importNo = getGlobalSetting('importNo') || 0;
   setGlobalSetting('importNo', importNo + 1);
-  const graphId = addGraph(
-    graphObj.graph.name + ' ' + importNo,
-    graphObj.graph
-  );
+  const graphId = addGraph(graphObj.graph.name + ' ' + importNo, {
+    ...graphObj.graph,
+    _id: uuid(),
```

Comment:
The reason is that I want to overwrite the existing _id in the file. Right now, addGraph only generates a new id if there isn't one already.